radix-common/src/constants/always_visible_nodes.rs

```diff
@@ -16,7 +16,7 @@ pub use radix_common::constants::*;
 // - Split bootstrapping into state flushing and transaction execution (the "chicken-and-egg" problem)
 //
 lazy_static! {
-    pub static ref ALWAYS_VISIBLE_GLOBAL_NODES: IndexSet<NodeId> = {
+    pub static ref ALWAYS_VISIBLE_GLOBAL_NODES_V1: IndexSet<NodeId> = {
```

Comment:
Now that we have the method, I think it's best to just always use the method to get access. So could we make these both not `pub`?

Reply:
Sounds good, these two do not need to be public.